L/LAMMPS/build_tarballs.jl

```diff
@@ -75,18 +75,19 @@ augment_platform_block = """
 # platforms are passed in on the command line
 # platforms = supported_platforms(; experimental=true)
 platforms = supported_platforms()
-platforms = expand_cxxstring_abis(platforms)
-
 platforms, platform_dependencies = MPI.augment_platforms(platforms)
 # Avoid platforms where the MPI implementation isn't supported
 # OpenMPI
-platforms = filter(p -> !(p["mpi"] == "openmpi" && arch(p) == "armv6l" && libc(p) == "glibc"), platforms)
+platforms = filter(p -> !(p["mpi"] == "openmpi" && nbits(p) == 32), platforms)
 # MPItrampoline
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && libc(p) == "musl"), platforms)
 platforms = filter(p -> !(p["mpi"] == "mpitrampoline" && Sys.isfreebsd(p)), platforms)
 
 platforms = filter(p -> !(Sys.isfreebsd(p) || libc(p) == "musl"), platforms)
 
+platforms = expand_cxxstring_abis(platforms)
+platforms = expand_gfortran_versions(platforms) # ABI dependent due to MPI
```

Comment:
We don't seem to have to do it for other packages which depend on MPI, see for example https://github.com/JuliaPackaging/Yggdrasil/blob/27db5ba5a624df4809faef80a12297ccbc570ab4/A/ADIOS2/build_tarballs.jl